exonum/src/node/state.rs

```diff
@@ -568,9 +568,10 @@ impl State {
     }
 
     /// Updates known round for a validator and returns
-    /// a new actual round if at least one non byzantine validators are on a higher round.
+    /// a new actual round if at least one non byzantine validators is guaranteed to be on a higher round.
     /// Otherwise returns None.
-    pub fn get_actual_round(&mut self, id: ValidatorId, round: Round) -> Option<Round> {
+    pub fn update_validator_round(&mut self, id: ValidatorId, round: Round) -> Option<Round> {
```

Comment:
btw: this method does not really "update" validator round

Reply:
It updates known round, as @slowli mentioned at the PR #680.